apis/openstack-heat/src/main/java/org/jclouds/openstack/heat/v1/domain/Attributes.java

```diff
@@ -0,0 +1,189 @@
+/**
+ * 
+ */
+package org.jclouds.openstack.heat.v1.domain;
+
+import java.beans.ConstructorProperties;
+import java.util.Set;
+
+import javax.inject.Named;
+
+import org.jclouds.openstack.v2_0.domain.Link;
+import org.jclouds.openstack.v2_0.domain.Resource;
+
+import com.google.common.base.Objects;
+import com.google.common.base.Objects.ToStringHelper;
+
+/**
+ * @author deviyada
+ * 
+ */
+public class Attributes extends Resource {
```

Comment:
please change the code style to be:
Tab size :3
Indent:3
Continuation indent:6